tests/receivers/postgresql/bundled_test.go

```diff
@@ -38,6 +38,41 @@ import (
 	"github.com/signalfx/splunk-otel-collector/tests/testutils/kubeutils/manifests"
 )
 
+func TestPostgresqlDockerObserver(t *testing.T) {
```

Comment:
similarly here do you mind copying instead of removing existing coverage? The previous test will need updating to enable the SA receiver and disable this one.

Reply:
Will do, was going to ask about that in last comment. Figured since you didn't mention it explicitly it wasn't something you were concerned about. 

Will get this change going. Thanks! 👍